src/interpreter/Interpreter.as

```diff
@@ -677,11 +677,11 @@ public class Interpreter {
 	// a reference to the Variable object is cached in the target object.
 
 	private function primVarGet(b:Block):* {
-		if (activeThread == null) return 0;
+		var target:ScratchObj = app.runtime.currentDoObj ? app.runtime.currentDoObj : activeThread.target;
```

Comment:
What do you think about:
1) Flipping this conditional: `var target:ScratchObj = activeThread ? activeThread.target : app.runtime.currentDoObj;`
and
2) Always setting `currentDoObj` in `allStacksAndOwnersDo`
?

The reason I ask is that I'm worried about the possibility of getting in here with both `activeThread` and `currentDoObj` unset.